maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -149,6 +151,7 @@ public CLIManager()
         options.addOption( Option.builder( LEGACY_LOCAL_REPOSITORY ).longOpt( "legacy-local-repository" ).desc( "Use Maven 2 Legacy Local Repository behaviour, ie no use of _remote.repositories. Can also be activated by using -Dmaven.legacyLocalRepo=true" ).build() );
         options.addOption( Option.builder( BUILDER ).longOpt( "builder" ).hasArg().desc( "The id of the build strategy to use" ).build() );
         options.addOption( Option.builder( NO_TRANSFER_PROGRESS ).longOpt( "no-transfer-progress" ).desc( "Do not display transfer progress when downloading or uploading" ).build() );
+        options.addOption( Option.builder().longOpt( COLOR ).hasArg().desc( "Color support mode (auto/always/never)" ).build() );
```

Comment:
Could you please be a bit more elaborate on what this flag entails? I fear not everyone will know what it will do.

Reply:
@MartinKanters This is a manifestation of the sytem property. No logical changes.

Reply:
I understand what it does, and I agree with adding the flag. My point was that people will find this option in `mvn help` and might not know what it entails. For example, `ls` has this as help text:
> --color[=WHEN] colorize the output; WHEN can be 'always' (default if omitted), 'auto', or 'never'; more info below

So perhaps we can change the description to something like:
```suggestion
        options.addOption( Option.builder().longOpt( COLOR ).hasArg().desc( "Defines the color mode of the output. Available options are auto/always/never" ).build() );
```

Reply:
The suggestion looks good to me.

Reply:
Alright, could you please take and squash it? Afterwards, I'll make sure to test it on Jenkins and take care of merging.

Reply:
@MartinKanters done